### PR TITLE
for #7584 - make about libraries toolbar theme consistent

### LINKED ISSUE #7584: [Bug] wrong header bar background color for libraries screen
### Steps to reproduce

1. Open Settings > About Firefox Preview > Libraries that we use

### Expected behavior

The header bar has the same background color as every other screen in Firefox Preview (white).

### Actual behavior

The header bar is black.

Screenshot_20200109-233559

### Device information

* Android device: OnePlus 7T Pro McLaren Edition / Android 10
* Fenix version: master branch revision 0a412a1a6ae218e3ea0e9f67604c37c5355eb61c

### LINKED ISSUE #13970: [Bug] First item in new about libraries is hidden
## Steps to reproduce
Open the new about libraries page.

### Expected behavior
All libraries are shown.

### Actual behavior
The first library is not shown. The first displayed one is `android arch-common` in nightly, but is `absl` in beta. Also, the scroll bar starts hidden behind the toolbar. I think `absl` is also hidden behind it. cc @gilbsgilbs 
### Device information

* Android device: ?
* Fenix version: ?

### PR DESCRIPTION
**Pull request by gilbsgilbs**
- fixes #7584 
- fixes #13970 

Before:

90272316-8604ae80-de5d-11ea-9137-035f6d0c6812
90272322-8735db80-de5d-11ea-8565-8d6631e16bf1

After:

2020-08-20T14:01:11+02:00
2020-08-20T14:01:20+02:00

### Pull Request checklist

- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not - very subtle UI change, hard to test
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by gilbsgilbs** [comment]
Actually, this is still not perfect as the back button is slightly off. I'll look into it.

**Comment by gilbsgilbs** [comment]
Should be good to merge :heavy_check_mark: .

File: app/src/main/java/org/mozilla/fenix/settings/about/AboutLibrariesFragment.kt [hunk 1]
```diff
@@ -27,33 +31,38 @@ import java.util.Locale
...
         val appName = getString(R.string.app_name)
-        title = getString(R.string.open_source_licenses_title, appName)
-        setContentView(R.layout.about_libraries_activity)
+        showToolbar(getString(R.string.open_source_licenses_title, appName))
 
-        setSupportActionBar(findViewById(R.id.toolbar))
-        supportActionBar?.setDisplayHomeAsUpEnabled(true)
-        supportActionBar?.setDisplayShowHomeEnabled(true)
+        setupLibrariesListView(
+            fragment.findViewById(R.id.about_libraries_listview)
```
Comment (NotWoods) [inline app/src/main/java/org/mozilla/fenix/settings/about/AboutLibrariesFragment.kt:53]: We use a Kotlin plugin so you can refer to views by ID directly:
```suggestion
            view.about_libraries_listview
```

File: app/src/main/java/org/mozilla/fenix/settings/about/AboutLibrariesFragment.kt [hunk 2]
```diff
@@ -27,33 +31,38 @@ import java.util.Locale
...
  * but we considered the risk of introducing such third-party dependency
  * to Fenix too high. Therefore, we use Google's gradle plugin to
- * extract the dependencies and their licenses, and this activity
+ * extract the dependencies and their licenses, and this fragment
  * to show the extracted licenses to the end-user.
  */
-class AboutLibrariesActivity : AppCompatActivity() {
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
+class AboutLibrariesFragment : Fragment() {
```
Comment (NotWoods) [inline app/src/main/java/org/mozilla/fenix/settings/about/AboutLibrariesFragment.kt:37]:
```suggestion
class AboutLibrariesFragment : Fragment(R.layout.about_libraries_fragment) {
```

File: app/src/main/java/org/mozilla/fenix/settings/about/AboutLibrariesFragment.kt [hunk 3]
```diff
@@ -27,33 +31,38 @@ import java.util.Locale
...
+    override fun onCreateView(
```
Comment (NotWoods) [inline app/src/main/java/org/mozilla/fenix/settings/about/AboutLibrariesFragment.kt:38]: `onViewCreated` is called immediately after and lets us just use the fragment constructor to set up the layout.
```suggestion
    override fun onViewCreated(
```

File: app/src/main/java/org/mozilla/fenix/settings/about/AboutLibrariesFragment.kt [hunk 4]
```diff
@@ -27,33 +31,38 @@ import java.util.Locale
...
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val fragment = inflater.inflate(
```
Comment (NotWoods) [inline app/src/main/java/org/mozilla/fenix/settings/about/AboutLibrariesFragment.kt:43]: If you decide to keep `onCreateView` this variable should be renamed
```suggestion
        val view = inflater.inflate(
```

File: app/src/main/java/org/mozilla/fenix/settings/about/AboutLibrariesFragment.kt [hunk 5]
```diff
@@ -27,33 +31,38 @@ import java.util.Locale
...
-        return true
-    }
-
-    private fun setupLibrariesListView() {
+    private fun setupLibrariesListView(listView: ListView) {
         val libraries = parseLibraries()
-        val listView = findViewById<ListView>(R.id.about_libraries_listview)
-        listView.adapter = ArrayAdapter(this, android.R.layout.simple_list_item_1, libraries)
+        listView.adapter = ArrayAdapter(
+            requireContext(),
```
Comment (NotWoods) [inline app/src/main/java/org/mozilla/fenix/settings/about/AboutLibrariesFragment.kt:62]: You can get context from views and avoid the null/error check
```suggestion
            listView.context,
```

File: app/src/main/res/layout/about_libraries_fragment.xml [hunk 6]
```diff
@@ -3,23 +3,16 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
```
Comment (NotWoods) [inline app/src/main/res/layout/about_libraries_fragment.xml:3]: Generally we put `fragment_` as a prefix, not suffix. Can you name the file `fragment_about_libraries`?

File: app/src/main/res/navigation/nav_graph.xml [hunk 7]
```diff
@@ -587,7 +587,15 @@
 
     <fragment
         android:id="@+id/aboutFragment"
-        android:name="org.mozilla.fenix.settings.about.AboutFragment" />
+        android:name="org.mozilla.fenix.settings.about.AboutFragment">
```
Comment (NotWoods) [inline app/src/main/res/navigation/nav_graph.xml:590]: Connect the layout file to improve the preview pane
```suggestion
        android:name="org.mozilla.fenix.settings.about.AboutFragment"
        tools:layout="@layout/fragment_about_libraries">
```

File: app/src/main/java/org/mozilla/fenix/settings/about/AboutFragment.kt [hunk 8]
```diff
@@ -167,8 +168,8 @@ class AboutFragment : Fragment(), AboutPageListener {
     }
 
     private fun openLibrariesPage() {
-        val intent = Intent(requireContext(), AboutLibrariesActivity::class.java)
-        startActivity(intent)
+        val navController = Navigation.findNavController(requireActivity(), R.id.container)
```
Comment (NotWoods) [inline app/src/main/java/org/mozilla/fenix/settings/about/AboutFragment.kt:171]:
```suggestion
        val navController = findNavController()
```

File: app/src/main/res/navigation/nav_graph.xml (hunk 7 above)
Comment (gilbsgilbs) [reply app/src/main/res/navigation/nav_graph.xml:590]: This one should be `fragment_about` right?

**Pull request merged**